backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/RTTIGenerator.kt

```diff
@@ -172,7 +172,8 @@ internal class RTTIGenerator(override val context: Context) : ContextUtils {
             floatType to 6,
             doubleType to 7,
             kInt8Ptr to 8,
-            int1Type to 9
+            int1Type to 9,
+            vectorType to 10
```

Comment:
Should be also supported in runtime debug support.
See [comment above](https://github.com/JetBrains/kotlin-native/blob/e1b3fefd3333fa730bb00174b7eb24c90822f6a0/backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/RTTIGenerator.kt#L165).